source/event.c

```diff
@@ -244,15 +245,15 @@ void eventOS_scheduler_set_active_tasklet(int8_t tasklet)
 {
     curr_tasklet = tasklet;
 }
-
+/*
```

Comment:
I think I've belatedly figured out why this is commented out - it doesn't link if EXCLUDE_HIGH_RES_TIMER is defined?

Yes, it looks like this needs a proper substitute implementation for the NS_EVENTLOOP_USE_TICK_TIMER case. Or is it a dummy in that case? Maybe @kuggenhoffen would like to check.